drivers/wdm/audio/backpln/portcls/interfaces.hpp

```diff
@@ -924,26 +924,26 @@ typedef IPortPinDMus *PPORTPINDMUS;
         IN  ULONG BufferSize,                                             \
         IN  PPHYSICAL_ADDRESS PhysicalAddressConstraint OPTIONAL);        \
                                                                           \
-    STDMETHODIMP_(void) FreeBuffer(void);                                 \
-    STDMETHODIMP_(ULONG) TransferCount(void);                             \
-    STDMETHODIMP_(ULONG) MaximumBufferSize(void);                         \
-    STDMETHODIMP_(ULONG) AllocatedBufferSize(void);                       \
-    STDMETHODIMP_(ULONG) BufferSize(void);                                \
+    STDMETHODIMP_(VOID) FreeBuffer(VOID);                                 \
+    STDMETHODIMP_(ULONG) TransferCount(VOID);                             \
+    STDMETHODIMP_(ULONG) MaximumBufferSize(VOID);                         \
+    STDMETHODIMP_(ULONG) AllocatedBufferSize(VOID);                       \
+    STDMETHODIMP_(ULONG) BufferSize(VOID);                                \
                                                                           \
-    STDMETHODIMP_(void) SetBufferSize(                                    \
+    STDMETHODIMP_(VOID) SetBufferSize(                                    \
         IN  ULONG BufferSize);                                            \
                                                                           \
-    STDMETHODIMP_(PVOID) SystemAddress(void);                             \
+    STDMETHODIMP_(PVOID) SystemAddress(VOID);                             \
     STDMETHODIMP_(PHYSICAL_ADDRESS) PhysicalAddress(                      \
         IN  PPHYSICAL_ADDRESS PhysicalAddressConstraint OPTIONAL);        \
-    STDMETHODIMP_(PADAPTER_OBJECT) GetAdapterObject(void);                \
+    STDMETHODIMP_(PADAPTER_OBJECT) GetAdapterObject(VOID);                \
                                                                           \
-    STDMETHODIMP_(void) CopyTo(                                           \
+    STDMETHODIMP_(VOID) CopyTo(                                           \
```

Comment:
Please revert all these "void" --> "VOID" changes of yours in this file, and in the corresponding implementation functions : Check official portcls.h (that defines the IDmaChannel interface amongst others), they use "void", so let's stay compatible there.

Reply:
Reverted.